assembl/templates/adminIndex.jinja2

```diff
@@ -0,0 +1,66 @@
+{% extends 'base.jinja2' %}
+
+{% block page_title %}{% trans %}Administration{% endtrans %}{% endblock %}
```

Comment:
Arf... trans is the proper jinja2 mechanism, gettext forces us to send gettext as a jinja param for no good reason. That pattern won somehow, and I don't see myself refactoring it.